src/webview/html/messageAsHtml.js

```diff
@@ -72,13 +77,87 @@ $!${messageReactionListAsHtml(reactions, ownUser.user_id, allImageEmojiById)}
 `;
 };
 
-const widgetBody = (message: Message | Outbox) => template`
+/**
+ * Render the body of a message that has submessages.
+ *
+ * Must not be called on a message without any submessages.
+ */
+const widgetBody = (message: Message, ownUserId: UserId) => {
+  invariant(
+    message.submessages !== undefined && message.submessages.length > 0,
+    'should have submessages',
+  );
+
+  const widgets = message.submessages
```

Comment:
This doesn't seem like the right name -- the whole thing is a widget. These submessages are things that happened to the widget.

Perhaps `widgetSubmessages`?

Reply:
Renamed :)###

Reply:
Please also propagate the rename to where we iterate through this array :) -- like here:
```js
  const events = widgetSubmessages.filter(widget => 'type' in widget.content);
```

Each of those is a submessage, and isn't a widget.

Reply:
Done. I also made a bunch of stuff camel case while I was at it :)